website/docs/guides/eks-getting-started.html.md

```diff
@@ -0,0 +1,594 @@
+---
+layout: "aws"
+page_title: "EKS Getting Started Guide"
+sidebar_current: "docs-aws-guide-eks-getting-started"
+description: |-
+  Using Terraform to configure AWS EKS.
+---
+
+# Getting Started with AWS EKS
+
+The Amazon Web Services EKS service allows for simplified management of
+[Kubernetes](https://kubernetes.io/) servers. While the service itself is
+quite simple from an operator perspective, understanding how it interconnects
+with other pieces of the AWS service universe and how to configure local
+Kubernetes clients to managed clusters can be daunting at first.
```

Comment:
While I might agree with your statement I'm not sure we want to put ourselves in the position of someone who can say what 3rd party service/product is or isn't _daunting_ in our docs generally. 🤔

Reply:
👍  I'll update the verbiage here to read `can be helpful.`